spec/bolt/node/ssh_spec.rb

```diff
@@ -13,6 +13,7 @@
   let(:user) { ENV['BOLT_SSH_USER'] || "vagrant" }
   let(:password) { ENV['BOLT_SSH_PASSWORD'] || "vagrant" }
   let(:port) { ENV['BOLT_SSH_PORT'] || 2224 }
+  let(:key) { ENV['BOLT_SSH_KEY'] || Dir[".vagrant/**/private_key"] }
```

Comment:
This looks like it will pass an array as `:key` in the default case. That doesn't reflect how this will actually work in practice.

Reply:
if we don't pass an array here tests may break if users switch vagrant providers.

Reply:
I'm ok just testing the string option in travis but if you want I can shell out to `vagrant provider linux` to find a single private key.

Reply:
I'm fine with this, knowing there's an actual reason it uses a glob.